src/components/tab-title/tab-title.e2e.ts

```diff
@@ -1,14 +1,48 @@
-import { newE2EPage } from "@stencil/core/testing";
+import { newE2EPage, E2EPage, E2EElement } from "@stencil/core/testing";
 import { disabled, HYDRATED_ATTR, renders, hidden } from "../../tests/commonTests";
+import { html } from "../../../support/formatting";
 import { CSS } from "./resources";
 
 describe("calcite-tab-title", () => {
   const tabTitleHtml = "<calcite-tab-title></calcite-tab-title>";
+  const tabTitleClosableHtml = "<calcite-tab-title closable></calcite-tab-title>";
+  const multiTabTitleClosableHtml = `
+    <calcite-tabs bordered position="top">
```

Comment:
I think it'd be beneficial to update the markup to use some sort of numbering in the IDs and/or content to make it easy to track the order in the tests below.

Reply:
Cool. This is better, but can be improved by also renaming the element reference variables below. 

For comparison, look at the following: https://github.com/Esri/calcite-components/blob/ee5444c6fd986c7ea87c0daf56054f64c13d525d/src/components/combobox/combobox.e2e.ts#L945-L968 You can see it's easier to visualize the order in the assertions without having to look at the HTML or selectors used.